app/assets/javascripts/admin/admin_ofn.js.coffee

```diff
@@ -12,5 +12,6 @@ angular.module("ofn.admin", [
   "admin.orders"
 ]).config ($httpProvider, $locationProvider, $qProvider) ->
   $httpProvider.defaults.headers.common["Accept"] = "application/json, text/javascript, */*"
+  # for the next line, you should also probably check file: app/assets/javascripts/admin/utils/utils.js.coffee
```

Comment:
This commit doesn't move or change anything, has it since been rebased? 

Still, the comment seems useful

Reply:
Yep: https://github.com/openfoodfoundation/openfoodnetwork/pull/10575#discussion_r1151916464